Sources/Steepshot/Steepshot.iOS/Views/FeedViewController.cs

```diff
@@ -199,6 +201,29 @@ private void HidePhoto(Post post)
             ShowAlert(error);
         }
 
+        private void CopyLink(Post post)
+        {
+            UIPasteboard clipboard = UIPasteboard.General;
+            clipboard.String = GetPostLink(post);
+
+            ShowAlert(LocalizationKeys.Copied);
+        }
+
+        private void SharePhoto(Post post)
+        {
+            var item = NSObject.FromObject(GetPostLink(post));
```

Comment:
It's worth avoiding using functions in the arguments of another function, this complicates the debugging of the application